src/UDS.Net.API/Data/Migrations/20240507153943_create-b9-cannabuse.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace UDS.Net.API.Data.Migrations
+{
+    /// <inheritdoc />
+    public partial class createb9cannabuse : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<int>(
+                name: "STRUCTAD",
+                table: "tbl_D1bs",
+                type: "int",
+                nullable: true,
+                comment: "7a1. Atrophy pattern consistent with AD",
+                oldClrType: typeof(int),
+                oldType: "int",
+                oldNullable: true,
+                oldComment: "7a1. Atrophy pattern conistent with AD");
+
+            migrationBuilder.AlterColumn<int>(
+                name: "IMAGEWMH",
+                table: "tbl_D1bs",
+                type: "int",
+                nullable: true,
+                comment: "7a3f. Extensive white-matter hyperintensity (CHS score 7-8+)",
+                oldClrType: typeof(int),
+                oldType: "int",
+                oldNullable: true,
+                oldComment: "7a3f. Extensive white-matter hyperinensity (CHS score 7-8+)");
```

Comment:
Looks like a comment attribute was adjusted since the last migration. Figured it was okay to leave this in this one instead of making 2 migrations for the 2 separate actions for now